### PR TITLE
[GHA] LBT skips on move-prover

### PR DESCRIPTION
**Pull request by sausagee**
## Motivation
Land-blocking test will skip on Move prover changes and other doc related changes. This will speed up the land.

## Test Plan
Testcase 1 - `language/move-prover/src/lib.rs` and `vm-validator/src/vm_validator.rs` are updated in https://github.com/sausagee/libra/commit/d5607c5db2c2a727321bb11e7047bd69fbb98f97 --> expect Action to trigger

Testcase 2 - with Testcase1 as parent, only `language/move-prover/stackless-bytecode-generator/src/stackless_bytecode.rs` is updated in https://github.com/sausagee/libra/commit/ccbde2797397e60a9339cbf77e94bd4d51ea2aa6  --> expect Action to NOT trigger

Testcase 3 - with Testcase2 as parent, only `network/src/peer/mod.rs` is update in https://github.com/sausagee/libra/commit/ca323f5aec882df4de55ceaf92b077f410557d49 --> expect Action to trigger.

Verified that Testcase1 and Testcase3 triggered Action but not Testcase2.
image

### CODE REVIEW

File: .github/workflows/land-blocking.yml [hunk 1]
```diff
@@ -5,6 +5,10 @@ on:
   push:
     branches:
       - auto
+    paths-ignore:
```
Comment (sausagee) [inline .github/workflows/land-blocking.yml]: Just realized that we config Homu to wait for signal from this workflow.  @metajack, do you know if Homu would get stuck in waiting or refuse to land the PR in this case?

Comment (metajack) [reply .github/workflows/land-blocking.yml]: Do you know exactly what happens? Does it just send success immediately, or does it not even show up on the PR at all? I couldn't find anything in the docs besides `the workflow will not run`.

Comment (sausagee) [reply .github/workflows/land-blocking.yml]: It doesn't seem to show up on the PR. Querying the actions/runs API doesn't yield a record of the PR.  I am not sure if there is something happening behind the scene beyond what I can tell.  I will ask in the GH Actions community. 

BTW, what event ("send success immediately") is Homu expecting?

Comment (metajack) [reply .github/workflows/land-blocking.yml]: homu gets webhook notifications for github checks API changes. So when an action adds a check to the PR or when a check status changes to succeeded or failed, etc.

Homu won't land PRs unless it gets a success status for every check it considers land blocking, so you're right that homu will get stuck if cluster-test is skipped.

An alternative to `paths-ignore` would be to do the paths-ignore filtering yourself in the GHA and then just trivially succeed if all paths match the ignore targets.

Comment (sausagee) [reply .github/workflows/land-blocking.yml]: Yeah, I was originally hoping the `paths-ignore` option would be a quick and dirty hack. DIY filter is probably better as we can use it in CircleCI as well.

**Comment by sausagee** [comment]
Going to close this for now. I will do the filtering elsewhere which it can be the single source of truth and can be shared with circleci's workflow.

**Pull request closed without merging**